### PR TITLE
[OPS-201] [WIP] log streaming

### PR DESCRIPTION
**Pull request by yorickvP**
Started writing some erlang. The http handler isn't there yet. This is pretty much my first erlang, is any of this a good idea?

### CODE REVIEW

**Comment by yorickvP** [comment]
Might want to use https://ninenines.eu/docs/en/cowboy/2.3/guide/loop_handlers/

File: src/nix_ci_logs.erl [hunk 1]
```diff
@@ -0,0 +1,39 @@
+-module(nix_ci_logs).
+-export([init/2]).
+-export([info/3]).
+
+-record(state, {pid, ref, monref, exists}).
```
Comment (lukateras) [inline src/nix_ci_logs.erl:5]: Map would be better.

File: src/nix_ci_logs.erl [hunk 2]
```diff
@@ -0,0 +1,39 @@
+-module(nix_ci_logs).
+-export([init/2]).
+-export([info/3]).
```
Comment (lukateras) [inline src/nix_ci_logs.erl:3]: Nitpick:

```
-export([init/2, info/3]).
```

**Review by lukateras — COMMENTED** [review]
This actually looks pretty good!

**Pull request closed without merging**